handler_success.go

```diff
@@ -75,6 +92,10 @@ func (s *SuccessHandler) RecordHit(r *http.Request, timing int64, code int, requ
 			alias = session.Alias
 		}
 
+		if len(s.Spec.TagHeaders) > 0 {
```

Comment:
I assume we should add tagging to error handler too.

Reply:
See the same code in [error_handler.go](https://github.com/TykTechnologies/tyk/pull/1139/files#diff-e0d46d3f416ee63c099afead1b191696R115)?

Reply:
my bad